components/checkbox/index.jsx

```diff
@@ -361,6 +386,82 @@ class Checkbox extends React.Component {
 		</div>
 	);
 
+	renderVisualPickerVariant = (props, assistiveText) => (
+		<span
+			className={classNames(
+				'slds-visual-picker',
+				`slds-visual-picker_${this.props.size}`,
+				this.props.vertical ? 'slds-visual-picker_vertical' : null
+			)}
+		>
+			<input
+				aria-controls={this.props['aria-controls']}
+				aria-describedby={this.props['aria-describedby']}
+				aria-owns={this.props['aria-owns']}
+				aria-required={this.props['aria-required']}
+				disabled={props.disabled}
+				/* A form element should not have both checked and defaultChecked props. */
+				{...(props.checked !== undefined
+					? { checked: props.checked }
+					: { defaultChecked: props.defaultChecked })}
+				id={this.getId()}
+				name={props.name}
+				onBlur={props.onBlur}
+				onChange={this.handleChange}
+				onFocus={props.onFocus}
+				onKeyDown={props.onKeyDown}
+				onKeyPress={props.onKeyPress}
+				onKeyUp={props.onKeyUp}
+				ref={(component) => {
+					this.input = component;
+				}}
+				role={props.role}
+				required={props.required}
+				type="checkbox"
+			/>
+			<label className="slds-checkbox_button__label" htmlFor={this.getId()}>
+				{this.props.coverable ? (
+					<div className="slds-visual-picker__figure slds-visual-picker__icon slds-align_absolute-center">
+						<span className="slds-is-selected">
+							{this.props.onRenderVisualPickerSelected}
+						</span>
+						<span className="slds-is-not-selected">
+							{this.props.onRenderVisualPickerNotSelected}
+						</span>
+					</div>
+				) : (
+					<span className="slds-visual-picker__figure slds-visual-picker__text slds-align_absolute-center">
+						{this.props.onRenderVisualPicker}
+					</span>
+				)}
+				{!this.props.vertical ? (
+					<span className="slds-visual-picker__body">
+							<span className="slds-text-heading_small">
+								{this.props.label}
```

Comment:
The prop `label` has been deprecated for checkbox. This will need to be `this.props.labels.label`